src/csharp/Grpc.Core/SerializationContext.cs

```diff
@@ -29,6 +31,9 @@ public abstract class SerializationContext
         /// payload which must not be accessed afterwards.
         /// </summary>
         /// <param name="payload">the serialized form of current message</param>
-        public abstract void Complete(byte[] payload);
+        public virtual void Complete(byte[] payload)
+        {
+            throw new NotImplementedException();
```

Comment:
same comment here as for the `abstract -> default-throws` comment on `DeserializationContext` above.

Reply:
Same as above